espnet2/bin/asr_inference.py

```diff
@@ -1,18 +1,22 @@
 #!/usr/bin/env python3
```

Comment:
@LiChenda, @Emrys365, and @ftshijt can you take a look at this code?
We also want to release the espnet-se/diarization models from huggingface hub and I want to ask you to do the same thing for `enh_inference.py` and `diar_inference.py`.

Reply:
Sure.